public/images/os/arch.svg

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Just copied them all from the current v1 setup. We'll do a sweep as we get closer to a release :)